app/services/notifications/reactions/reaction_data.rb

```diff
@@ -26,10 +26,11 @@ def self.coerce(object)
         when Notifications::Reactions::ReactionData
           object
         when Reaction
+          user_id = object.reactable.user_id unless object.reactable_type == "User"
```

Comment:
_this_ check could be removed if I aliased User#user_id to User#id.

The assignment here is the fastest way not to raise a no method error, we'll raise a data error later from new.